Package.swift

```diff
@@ -23,6 +23,10 @@ let package = Package(
                       checksum: "85e573e369b28b6c6829c8925a2629e62966c8f22c659863a3005f458e9bf026"),
```

Comment:
The title reads 2.28.0 but the url 2.27.1.
I imagine the reason is to avoid breaking current SPM package but during 2.28.0 release this value will be updated, correct?